frontend/src/App.vue

```diff
@@ -1,16 +1,18 @@
 <script setup lang="ts">
```

Comment:
Use script tag below template. Structure code like this:
1. template tag
2. script tag
3. style tag